nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/CustomerRightsRetentionStrategy.java

```diff
@@ -0,0 +1,30 @@
+package no.unit.nva.model.associatedartifacts;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import nva.commons.core.JacocoGenerated;
+
+@JsonTypeInfo(use = Id.NAME, property = "type")
+@JsonTypeName(CustomerRightsRetentionStrategy.TYPE_NAME)
+public class CustomerRightsRetentionStrategy implements RightsRetentionStrategy {
+    private static final int STATIC_VALUE_FOR_HASH_CODE = 88_961;
```

Comment:
Use a random unique value here…i.e. 982_114

Reply:
While there is no real risk of confusion, in most cases, I think it's a good idea not to re-use the value between objects.